bitbot/tarot.go

```diff
@@ -0,0 +1,117 @@
+package bitbot
+
+import (
+	"github.com/whyrusleeping/hellabot"
+	"math/rand"
+	"strings"
+	"fmt"
+)
+
+var TarotTrigger = NamedTrigger{
+	ID:   "Tarot",
+	Help: "Request tarot cards, default 1. Usage !tarot [num cards].",
+	Condition: func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "PRIVMSG" && strings.HasPrefix(m.Trailing, "!tarot")
+	},
+	Action: func(irc *hbot.Bot, m *hbot.Message) bool {
+		if (len(m.Content) > 7) {
+			resp := tarotCards[rand.Intn(len(magic8responses)-1)]
+			irc.Reply(m, resp)
+	} else {
+		msg := strings.TrimPrefix(m.Content, "!tarot ")
+        num := 1
+		fmt.Sscanf(msg, "%d", &num)
+		if (num < 1 || num > 78) {
+			num = 1
+		}
+		deck := rand.Perm(77)
```

Comment:
Try to avoid magic numbers. This looks like it's calculated based on the length of the deck. Why not use that instead?